Main.js

```diff
@@ -0,0 +1,10 @@
+const gistUpload = require('./gistUpload.js');
+
+const gistName = process.argv[3];
+const file = process.argv[2];
+
+function Runner() {
+  gistUpload.readFile(gistName, file);
```

Comment:
assign this to something and then console log the output rather than console logging in either function. Have those functions just return a string